Shifting a 32-bit integer by 32 or more is undefined behavior

--- a/src/tests.cpp
+++ b/src/tests.cpp
@@ -3934,7 +3934,7 @@
 	if (sig_bits > MAX_SIG_BITS) issue_error();
 	if (exp_bits > 6) issue_error();
 	for (int i = 0; i <= 32; i++) {
-		Uint32 x = i, y = 1 << i;
+		Uint32 x = i, y = 1LL << i;
 		if (x >= 32) { x = 32; y = 0; }
 		if (count_low_zeroes32(y) != x) issue_error("count_low_zeroes32 behaving incorrectly");
 	}
